src/components/CommentList.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Comment from './Comment';
+
+function CommentList({ postId, comments }) {
+  const FilterComments = comments.filter(comment => comment.postId === postId);
```

Comment:
```suggestion
  const filteredComments = comments.filter(comment => comment.postId === postId);
```
Почему ты не передаешь комментарии сразу с постом, зачем передавать все комментарии и в каждом посте запускать filter?